src/test/java/com/redhat/engineering/srcclr/internal/SecurityDataProcessorTest.java

```diff
@@ -242,7 +237,7 @@ private Object executeMethod( Object instance, String name, Object[] params ) th
         for ( int i = 0; i < params.length; i++ )
             types[i] = params[i].getClass();
 
-        Method m = c.getDeclaredMethod( name, types );
+        Method m = c.getDeclaredMethod( "lookUpAPI", types );
```

Comment:
This change is also fine. Just to clarify, the reason for getting the method name parameter was that it was to be used for multiple tests, although there's only one test at the moment. Maybe I could change it back later if more test cases are needed.

Reply:
Ah ok. Yes that makes sense to me :-)